python/gui/auto_generated/qgsmaptoolcapture.sip.in

```diff
@@ -213,14 +213,16 @@ Number of points digitized
 :return: Number of points
 %End
 
-    QVector<QgsPointXY> points() const;
+    QgsPointSequence points() const;
```

Comment:
This is still an API break, right?

Reply:
Yes, and unfortunately I can't overload this method. That was already my problem in https://github.com/qgis/QGIS/pull/30108 So I created a new one pointsZM (don't know if it's a good name).